resources/views/partials/item/wishlist-login.blade.php

```diff
@@ -0,0 +1,3 @@
+<template v-if="!$root.loggedIn">
+    @lang('You must log in to add a product to your Order List.') <a href="/login" class="text-ct-accent underline underline-offset-3">@lang('login')</a>
```

Comment:
`Order List` capitalized? Seems kinda weird to me, this isn't german where every noun is capitalized.